src/main/java/levy/art/json/ArtObject.java

```diff
@@ -1,13 +1,32 @@
 package levy.art.json;
 
-import com.google.gson.annotations.SerializedName;
-import levy.art.json.ArtImage;
 
 public class ArtObject {
-    public String title;
-    public String longTitle;
-    public ArtImage webImage;
-    public String principalOrFirstMaker;
+    private String title;
+    private String principalOrFirstMaker;
+    private WebImage webImage;
 
-}
+    public String getTitle() {
+        return title;
+    }
+
+    public String getArtist() {
+        return principalOrFirstMaker;
+    }
+
+    public String getImageUrl() {
+        return webImage != null ? webImage.getUrl() : null;
+    }
 
+    public WebImage getWebImage() {
+        return webImage;
+    }
+
+    public static class WebImage {
```

Comment:
Why did you replace ArtImage with WebImage?

Reply:
I just felt it better match the json name